product_docs/docs/biganimal/release/free_trial/detail/experiment/backup_and_restore.mdx

```diff
@@ -100,13 +100,13 @@ For the free trial, you're limited to one active cluster. Restoring a cluster cr
     For more on using the BigAnimal CLI, see [Creating a cluster on the command line](../create_a_cluster/create_cluster_cli) and the [BigAnimal CLI reference](/biganimal/latest/reference/cli/).
 
 ```shell
-biganimal delete-cluster --id p-xxxxxxxxxxx
+biganimal cluster delete --id p-xxxxxxxxxxx
 ```
 
 Restore the cluster using the timestamp before the time you dropped the table.
 
 ```shell
-biganimal restore-cluster --id p-xxxxxxxxxxx --from-deleted --restore-point 2022-03-31T02:39:40+0000
+biganimal cluster restore --id p-xxxxxxxxxxx --from-deleted --restore-point 2022-03-31T02:39:40+0000
 __OUTPUT__
```

Comment:
I wasn't able to verify this output because I wasn't able to connect to my cluster.